mule-extensions-api/src/main/java/org/mule/runtime/extension/api/loader/ExtensionLoadingContext.java

```diff
@@ -124,6 +125,12 @@ public interface ExtensionLoadingContext {
    */
   boolean isOCSEnabled();
 
+  /**
+   * @return whether components that have been disabled with the {@link Ignore} directive are to be included
+   * @since 1.5
+   */
+  boolean isEnableIgnoredComponents();
```

Comment:
I don't want to APIfy this. Set it through `org.mule.runtime.extension.api.loader.ExtensionLoadingContext#addParameter(String, Object)` instead